src/gen_dragonfruit/gd_dev_spanel.cpp

```diff
@@ -5,13 +5,11 @@ void gd_dev_spanel_open(void){
 }
 
 int gd_dev_spanel_read(void){
-    int value = 555;
-
+    float value = 555.0;
     #ifndef SEN_STUB
-    value = analogRead(_PIN_GD_SPANEL_);
-    //2*analogRead(_PIN_GD_SPANEL_)*5000.0/1023;
+    // Voltage divider on board halves the reading, so multiply by two to obtain orginal reading. 
```

Comment:
This could still be worded better: why do we have the voltage divider there in the first place? Although this is documented by the hardware team possibly, it's useful information to have in the driver anyways.